index.html

```diff
@@ -0,0 +1,331 @@
+            </div>
+          </div>
+          <div class="speaker-member">
+            <img src="image/speaker_03.png" alt="" class="member" />
+            <div class="spk-flex">
+              <h4 class="spk-name">Lila tretikov</h4>
+              <p class="spk-red">
+                Executive Director of the Wikimedia Foundation
+              </p>
+              <p class="spk-space">_____</p>
+              <p class="spk-black">
+                Lila Tretikov is the Executive of the Wikimedia Foundation, the
+                nonprofit organization that operates Wikipedia. Wikipedia is
+                freely available in 290 languag-es and used by nearly half a
+                billion people around the world every month.
+              </p>
+            </div>
+          </div>
+          <div class="speaker-member">
+            <img src="image/speaker_06.png" alt="" class="member" />
+            <div class="spk-flex">
+              <h4 class="spk-name">Ryan Merkley</h4>
+              <p class="spk-red">
+                CEO of Creativve Commons, ex COO of the Mozilla Foundation
+              </p>
+              <p class="spk-space">_____</p>
+              <p class="spk-black">
+                Ryan had been leading open-source projects at the Mozilla
+                Foundation such as the open-source move-ment
+              </p>
+            </div>
+          </div>
+        </div>
+      </aside>
+      <aside class="som">
+        <div class="speaker-container">
+          <div class="speaker-member">
+            <img src="image/speaker_01.png" alt="" class="member" />
+            <div class="spk-flex">
+              <h4 class="spk-name">Yochai Benkler</h4>
+              <p class="spk-red">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="spk-space">_____</p>
+              <p class="spk-black">
+                Benkler studies commons-based peer production, and published his
+                seminal book, The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+        
+        <div class="speaker-member ">
+          <img src="image/speaker_02.png" alt="" class="member" ber/>
+          <div class="spk-flex">
+            <h4 class="spk-name">SohYeong Noh</h4>
+            <p class="spk-red">
+              Director of Art Centre Nabi and a board member of CC Korea
+            </p>
+            <p class="spk-space">_____</p>
+            <p class="spk-black">
+              As the main venue for new media art production in Korea, Nabi
+              promotes cross-disciplinary collaboration and understanding among
+              science technology, humanities, and the arts.
+            </p>
+          </div>
+        </div>
+      </div>
+        <button class="more">
+          MORE <img src="image/arrow_down 1.png" alt="" />
+        </button>
+      </aside>
+    </section>
```

Comment:
- [ ] `required`
- [ ] Great Job here as well but😞 
- [ ] This section `Featured speakers` should be created dynamically in JavaScript.
- [ ] You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
- [ ] Please make amends. 🥂